### PR TITLE
Update driver configuration verbiage for clarity

### PR DESCRIPTION
**Pull request by zraineri**

### CODE REVIEW

**Comment by bigmontz** [comment]
Hello, @zraineri. 

Sorry for the late response. I saw your SLA agreement, but it was missing the github username and email. Could you please sign it again?

Thanks

**Comment by zraineri** [comment]
> Hello, @zraineri.
> 
> Sorry for the late response. I saw your SLA agreement, but it was missing the github username and email. Could you please sign it again?
> 
> Thanks

Just updated it!

Thanks

File: packages/neo4j-driver/src/index.js [hunk 1]
```diff
@@ -152,8 +152,8 @@ const {
  *       //
  *       // Default value for this option is `false` for backwards compatibility.
  *       //
- *       // **Warning:** `BigInt` doesn't implement the method `toJSON`. In maner of serialize it as `json`, It's needed to add a custom implementation of the `toJSON` on the
- *       // `BigInt.prototype` {@see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json}
+ *       // **Warning:** `BigInt` doesn't implement the method `toJSON`. In order to serialize it as `json`, a custom implementation of the method`toJSON` is
```
Comment (AndyHeap-NeoTech) [inline packages/neo4j-driver/src/index.js:155]: Space missing. Should be _method 'toJSON'_

**Review by AndyHeap-NeoTech — APPROVED** [review]

**Review by bigmontz — APPROVED** [review]
Hey @zraineri,

The PR looks fine and almost go to go. You should just address the comment made by @AndyHeap-NeoTech.

Thanks for contributing.

File: packages/neo4j-driver/src/index.js (hunk 1 above)
Comment (zraineri) [reply packages/neo4j-driver/src/index.js:155]: Nice catch

**Pull request closed without merging**